core/src/main/java/org/apache/accumulo/testing/core/randomwalk/security/Validate.java

```diff
@@ -101,12 +101,6 @@ public static void validate(State state, RandWalkEnv env, Logger log) throws Exc
       }
 
     }
-
-    Authorizations accuAuths = conn.securityOperations().getUserAuthorizations(WalkingSecurity.get(state, env).getTabUserName());
-    Authorizations auths = WalkingSecurity.get(state, env).getUserAuthorizations(WalkingSecurity.get(state, env).getTabCredentials());
-
-    if (!auths.equals(accuAuths))
-      throw new AccumuloException("Table User authorizations out of sync");
```

Comment:
After removing the inheritance from WalkingSecurity, the call here to get auths was now an error.  It then became obvious that these 2 calls to getUserAuthorizations were doing the same thing.